r_summary_stats/r_summary_stats.md

```diff
@@ -0,0 +1,810 @@
+    ##       28       30        6
+```    
+
+To get counts of two categorical variables together (e.g. what is the
+diagnosis type breakdown by recipient CMV status?), use
+[cross-tabulation](https://www.khanacademy.org/math/statistics-probability/analyzing-categorical-data/two-way-tables-for-categorical-data/a/two-way-tables-review), also called “crosstabs” or "two-way frequency tables".
+
+``` r
+xtabs(~cmv + diagnosis.type, data = cytomegalovirus)
+```
+```
+    ##      diagnosis.type
+    ## cmv   Lymphoid Myeloid
+    ##   No        15      19
+    ##   Yes       13      11
+```
+
+To get the results of a chi-squared test on the crosstabs, you can apply
+the `summary` function to an xtabs object.
+There are two equivalent ways to do that:
+
+``` r
+# you can nest the summary and xtabs functions
+summary(xtabs(~cmv + diagnosis.type, data = cytomegalovirus))
+```
+```
+    ## Call: xtabs(formula = ~cmv + diagnosis.type, data = cytomegalovirus)
+    ## Number of cases in table: 58 
+    ## Number of factors: 2 
+    ## Test for independence of all factors:
+    ##  Chisq = 0.569, df = 1, p-value = 0.4507
+```
+
+``` r
+# or you can save the xtabs object and then run summary() on it
+cmv_and_diagnosistype <- xtabs(~cmv + diagnosis.type, data = cytomegalovirus)
+summary(cmv_and_diagnosistype)
+```
+```
+    ## Call: xtabs(formula = ~cmv + diagnosis.type, data = cytomegalovirus)
+    ## Number of cases in table: 58 
+    ## Number of factors: 2 
+    ## Test for independence of all factors:
+    ##  Chisq = 0.569, df = 1, p-value = 0.4507
+```
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For beautifully printed crosstabs, see [the `tbl_cross` function in the `gtsummary` package](https://www.danieldsjoberg.com/gtsummary/articles/tbl_summary.html#tbl_cross).
+
+</div>
+
+### Coding Challenge 2
+ 
+ **Your turn!**
+
+ For your next coding challenge, return to RStudio and work until you see the next message telling you to stop and return to the module.
+
+
+## Presenting Summary Statistics in R
+
+Many research articles begin with a table summarizing the patients or research subjects analyzed, usually broken down by condition or exposure to allow the reader to see any baseline differences between groups. 
+For example, here is Table 1 from the paper this dataset is from: 
+
+![Table 1 summary statistics on the cytomegalovirus data.](media/cytomegalovirus_table1.png)
+
+
+One way to create a summary table would be to copy-paste all of your
+summary stats from R into whatever program you use to write your
+article. 
+That is both error-prone and tedious, and it also means if
+anything changes in your data you need to re-do all of that work.
+
+Another approach is creating the whole table right in R! 
+The `gtsummary` package is a great way to do that.
+
+### Creating Table 1 with `gtsummary`
+
+We’ll recreate an approximation of the cytomegalovirus Table 1 using `gtsummary`.
```

Comment:
Ooo thank you! I don't know how those got in there? I must have had something weird happen between vscode and rstudio at some point, since I know I worked on this doc in both programs.

Reply:
I did a search and replace and I think I got em all!